animation-box.html

```diff
@@ -48,10 +58,51 @@
        * @type string
        * @default ''
        */
-      transitions: ''
+      selected: 0,
+      transitions: 'hero-transition',
```

Comment:
The comment above is related with JS doc, we should put the comment above `trasitions`